packages/oui-pagination/src/pagination.html

```diff
@@ -47,30 +49,54 @@
                 ng-bind="page"
                 ng-click="$ctrl.onPageChange(page)"></button>
         </div>
-        <oui-dropdown ng-if="$ctrl.pageCount > 5">
+        <!-- /Buttons (<= 5 pages) -->
+
+        <!-- Dropdown (> 5 && <= 100 pages) -->
+        <oui-dropdown ng-if="$ctrl.pageCount > 5 && $ctrl.pageCount <= 100"
+            align="end">
             <button oui-dropdown-trigger
                 type="button"
                 ng-attr-aria-label="{{ $ctrl.translations.currentPageOfPageCount }}"
                 class="oui-button oui-button_dropdown oui-dropdown__trigger">
                 <span ng-bind="$ctrl.translations.currentPageOfPageCount"></span>
                 <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
             </button>
-            <oui-dropdown-content>
-                <div role="menu" class="oui-pagination-menu">
-                    <div class="oui-pagination-menu__items-list">
-                        <button type="button"
-                            ng-repeat="page in $ctrl.pageRange track by page"
-                            class="oui-pagination-menu__item"
-                            ng-attr-aria-label="{{ ::$ctrl.getPageAriaLabel(page) }}"
-                            ng-disabled="page === $ctrl.currentPage"
-                            ng-bind="page"
-                            ng-click="$ctrl.onPageChange(page)"></button>
-                    </div>
+            <oui-dropdown-content class="oui-pagination-menu">
+                <div class="oui-pagination-menu__items-list">
+                    <button type="button"
+                        ng-repeat="page in $ctrl.getPageRange() track by page"
+                        class="oui-pagination-menu__item"
+                        ng-attr-aria-label="{{ ::$ctrl.getPageAriaLabel(page) }}"
+                        ng-disabled="page === $ctrl.currentPage"
+                        ng-bind="page"
+                        ng-click="$ctrl.onPageChange(page)"></button>
                 </div>
             </oui-dropdown-content>
         </oui-dropdown>
-        <button type="button" class="oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
-            type="button"
+        <!-- /Dropdown (> 5 && <= 100 pages) -->
+
+        <!-- Numeric (> 100 pages) -->
+        <form class="oui-pagination-numeric"
+            ng-if="$ctrl.pageCount > 100"
+            ng-submit="$ctrl.checkPageRange($ctrl.inputPage)"
+            novalidate>
+            <span aria-hidden="true"
+                ng-bind="$ctrl.translations.inputPageOfPageCount[0]">
+            </span>
+            <input class="oui-pagination-numeric__input oui-input oui-input_xs"
```

Comment:
No `label` ?

Reply:
No need here. It render something like:

`Page [input] of x`

It's the reason I put the aria-hidden on the `span`, and a full label on the input.

Reply:
Seems like a nice pattern